week-2/implement/cases.js

```diff
@@ -15,3 +15,12 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+function capitalise(str){
+    words = str.split(" ");
+    camel = words.map(word => word.toUpperCase());
+    camelsnake = camel.join("_");
+  return camelsnake;
+}
+console.log(capitalise("lord of the rings"));
+console.log(capitalise("the great gatsby"));
+console.log(capitalise("the da vinci code"));
```

Comment:
This is really good, welldone, however I would use Keywords to avoid my variables treated as global for example:  const words = str.split(" ");
  const camel = words.map(word => word.toUpperCase());
  const camelsnake = camel.join("_");